src/main/webapp/statistics.html

```diff
@@ -152,10 +158,24 @@ <h2>Procedures</h2>
             very little data on them
           </li>
         </ul>
-    </ul>
+        <script>
```

Comment:
Would it maybe be better if the javascript was moved to another file?

Reply:
There are actually multiple blocks of javascript in the same file. Any reason why they aren't consolidated?

Reply:
That's a good point! I moved all of the javascript for the charts into a new file and I'll work on moving the collapsible as well.